teaspoons-cli/teaspoons/config.py

```diff
@@ -8,15 +8,36 @@
 class CliConfig:
```

Comment:
I think this is a perfectly good way to handle config

Reply:
thank you!